src/Wpf/Prism.Wpf/Modularity/IModuleCatalogExtensions.cs

```diff
@@ -116,34 +120,33 @@ public static IModuleCatalog AddModule(this IModuleCatalog catalog, string modul
         }
 
         /// <summary>
-        /// Adds the module.
+        /// Adds the module to the <see cref="IModuleCatalog"/>.
         /// </summary>
-        /// <param name="catalog">Catalog</param>
-        /// <param name="mode"><see cref="InitializationMode"/></param>
+        /// <param name="catalog">The catalog to add the module to.</param>
+        /// <param name="mode">Stage on which the module to be added will be initialized.</param>
```

Comment:
the updated mode description is really incorrect. It would be the `<see cref="InitializationMode"/>`